### ISSUE TITLE
A page jump on closing the lightbox

### ISSUE THREAD

**Issue by wetterkrank** [FreeFeed/freefeed-react-client#1183]
Chrome latest/iOS (12.1, 13.5) -- iPhone 7, iPhone SE2, iPhone 8 Plus at lambdatest.com.
Doesn't reproduce with Safari and Firefox/iOS, Chrome/Android or Chrome/Windows.

Can be reproduced with freefeed-react-client ran locally; freefeed.net (less reliably, may require 3-4 retries).

Steps to reproduce: see videos
https://www.dropbox.com/s/22qkxemrj137k9f/03-08-2020%2018-27-48.mkv?dl=0
https://www.dropbox.com/s/q0jyy8iec8jhp59/IMG_3158.MOV?dl=0

Expected behavior:
After opening and closing the lightbox, scroll position remains unchanged.

Actual behavior:
Page is scrolled to a seemingly random position.

**Comment by wetterkrank** [comment]
What I found out so far. If we change the following in **returnToOriginal()** function in _photoswipe.js_, this prevents the jump:
```
- history.back();
+ history.pushState('', document.title,  _windowLoc.pathname + _windowLoc.search );
```
However, that's still a workaround and a 2nd level dependency, so I'm not sure it will make a proper fix.

**Comment by davidmz** [comment]
Fixed in #1226
